shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/main/java/org/apache/shardingsphere/mode/manager/cluster/ClusterContextManagerBuilder.java

```diff
@@ -149,10 +147,10 @@ private SchemaConfiguration createSchemaConfiguration(final String schemaName, f
         return new DataSourceProvidedSchemaConfiguration(dataSources, schemaRuleConfigs);
     }
     
-    private void persistMetaData(final MetaDataPersistService metaDataPersistService, final Map<String, ShardingSphereSchema> schemaMap) {
-        for (Entry<String, ShardingSphereSchema> entry : schemaMap.entrySet()) {
-            metaDataPersistService.getSchemaMetaDataService().persist(entry.getKey(), entry.getValue());
-        }
+    private void persistMetaData(final MetaDataPersistService metaDataPersistService, final Map<String, ShardingSphereDatabase> databaseMap) {
+        databaseMap.entrySet().forEach(entry ->
+                entry.getValue().getSchemas().entrySet().forEach(schema ->
+                        metaDataPersistService.getSchemaMetaDataService().persist(schema.getKey(), schema.getValue())));
```

Comment:
invoke an interface in a loop, why not design another signature which support Map from very beginning

Reply:
I want to get value by ZooKeeper api, but it 's not supprot batch query.